chi/network.py

```diff
@@ -412,28 +414,35 @@ def get_router_id(name) -> str:
     return _resolve_id('routers', name)
 
 
-def create_router(router_name, gateway=False) -> dict:
-    """Create a router, with or without a public gateway.
-
-    Args:
-        router_name (str): The new router name.
-        gateway (bool): Whether to add a gateway to the public Internet on
-            the router. If no public gateway is requested, subnets connected
-            to this router will not have NAT to the Internet. (Default False).
-
-    Returns:
-        The created router representation.
-    """
-    router = {
-        'name': router_name,
-        'admin_state_up': True,
-    }
-
-    if gateway:
-        public_net_id = get_network_id(PUBLIC_NETWORK)
-        router['external_gateway_info'] = {'network_id': public_net_id}
-
-    return neutron().create_router(body=router)
+def create_router(router_name, gw_network_name=None) -> dict:
+    ''' 
+    Create a router with or without a public gateway. 
+    
+    Parameters
```

Comment:
Please avoid deviating from the existing docstring styles, we use [Google's Python docstring style](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings). Thanks however for updating docs :) 🏆

Reply:
Apologies - I updated this function by cross-referencing with an older version of chi. It looks like I copied over the wrong docs.